### ISSUE TITLE
Member Browser not loading in members

### ISSUE THREAD

**Issue by mahoneyl** [halcyon-tech/vscode-ibmi#180]
**Describe the bug**
I have a source file on our system we would like to load the members and edit them from. Previously this has worked flawlessly. This morning I went to open the source physical file and it returns an error on the GUI saying "Error loading members".
![image](https://user-images.githubusercontent.com/35392006/121717783-30158d00-caa7-11eb-910f-36fdbd8dac4d.png)
I am able to load members of smaller, personal library/sourcePF just fine. But when I try to load say a production library with many members. It stops working :(. 

**To Reproduce**
I'm scared that this might be a bug unique to our environment?
1. Go to Member Browser
2. Click on Add Source File (Add a Source File with at least 2743 members)
3. See error

**Expected behavior**
Source members should load in the drop down list

**Environment (please complete the following information):**
 - Extension version .5.5
 - IBM i OS version 7.2

**Additional context**
Thanks for your hard work!!!

**Comment by mahoneyl** [comment]
Are there some error logs or config logs I can pull somewhere and upload to be more helpful?

**Comment by worksofliam** [comment]
@mahoneyl There is an output channel called `Code for IBM i`.. open that up and then try to load the members again.

**Comment by mahoneyl** [comment]
Thanks for that tip I think something has changed between version 0.5.3 and 0.5.5 which has caused there to be errors when loading members.

I think the issue for our system comes from the way the source members were being pulled. It looks like in 0.5.3 that DSPFD was used to produce an outfile

In the newest version 0.5.5 the method of grabbing the source members using a sequel statement.
`/: DB2UTIL_JSON_CONTAINER=array /QOpenSys/pkgs/bin/db2util -o json "
        SELECT
          (b.avgrowsize - 12) as MBMXRL,
          a.iasp_number as MBASP,
          b.system_table_member as MBNAME,
          b.source_type as MBSEU2,
          b.partition_text as MBMTXT
        FROM qsys2.systables AS a
          JOIN qsys2.syspartitionstat AS b
            ON b.table_schema = a.table_schema AND
              b.table_name = a.table_name
        WHERE
          a.table_schema = 'MyLRGLib' And
          a.table_name = 'QRPGLESRC'
      "`

And then it looks like we got a JSON response object back. It looks like it actually got back a bunch of our source member information in the JSON Object. About 330602 characters in the stdout field.

**Comment by worksofliam** [comment]
Do you have SQL disabled in the settings by default? The issue is that SQL is always enabled right now, even if you have it disabled. 

I will look into getting that fixed.

**Comment by worksofliam** [comment]
@mahoneyl 0.5.6 is being deployed. I was able to recreate it.

Please update in a little bit and let me know how it goes.

**Comment by mahoneyl** [comment]
SQL was Enabled in my settings. 

> Do you have SQL disabled in the settings by default? The issue is that SQL is always enabled right now, even if you have it disabled. 
Was able to reproduce that same behavior. Never fell back to DSPFD

Thank you so much for jumping on  0.5.6 deployment you are so fast

**Comment by worksofliam** [comment]
@mahoneyl Was it enabled before 0.5.3? If it was using DSPFD I would probably say it was disabled.. 0.5.5 kind of broke that.

When you update I would recommend:

1. Connecting to your system
2. Go to the settings, disable SQL and save
3. Disconnect
4. Reconnect again 
5. Load the members

**Comment by mahoneyl** [comment]
I'm sorry but I can't remember for sure if I had enabled it purposefully or not before 0.5.5. 

I just tried out 0.5.6 with SQL disabled and its back to using DSPFD and loading the source members again. 💪Thank you!

One thing I am curious from when SQL was enabled; was why did large source pf get errors pulling but smaller ones did not? I'm wondering if we need to install db2utils or perhaps need to update it if we do already have it. I will check into seeing if it's loaded on our system.
